otx/algorithms/common/adapters/mmcv/utils.py

```diff
@@ -0,0 +1,101 @@
+"""Utils for common OTX algorithms."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import copy
+import tempfile
+from typing import Union
+
+from mmcv import Config, ConfigDict
+
+from otx.api.entities.datasets import DatasetEntity
+from otx.api.utils.argument_checks import (
+    DatasetParamTypeCheck,
+    check_input_parameters_type,
+)
+
+
+@check_input_parameters_type()
+def remove_from_config(config: Union[Config, ConfigDict], key: str):
+    """Update & Remove configs."""
+    if key in config:
+        if isinstance(config, Config):
+            del config._cfg_dict[key]  # pylint: disable=protected-access
+        elif isinstance(config, ConfigDict):
+            del config[key]
+        else:
+            raise ValueError(f"Unknown config type {type(config)}")
+
```

Comment:
Is there a reason for excepting prepare_for_training function?

Reply:
This is configured differently for each task, so I decided to leave it as it is.

Reply:
It seems that prepare_for_training function in otx/algo/det/utils/config_utils.py is quite general. 
Therefore we can add it this common directory

Reply:
thank you for comment! i will check it!

Reply:
@jaegukhyun The content related to the adaptive dataset is currently different from detection and segmentation, so it is not appropriate to move it to common.. adaptive repeat should be removed from seg, but that hasn't happened yet.
so... for now I left it as is

Reply:
I think common prepare_for_training function, which do not have adaptive function inside, would be added to common config_utils.py 
Let's add that following PR

Reply:
yes I think the same. Consider removing the adaptive repeat dataset from segmentation later and moving it to common!